pyccel/commands/pyccel_make.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+# coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/devel/LICENSE for full license details.      #
+#------------------------------------------------------------------------------------------#
+"""
+File containing the pyccel-make command line interface.
+"""
+
+import argparse
+import glob
+import os
+import sys
+from pathlib import Path
+
+def pyccel_make_command() -> None:
+    """
+    Pyccel console command.
+
+    The command line interface allowing pyccel-make to be called.
+    """
+    parser = argparse.ArgumentParser(description="Pyccel's project command line interface.",
+            add_help = False)
+
+    #... Help and Version
+    import pyccel
+    version = pyccel.__version__
+    libpath = pyccel.__path__[0]
+    python  = f'python {sys.version_info.major}.{sys.version_info.minor}'
+    message = f'pyccel {version} from {libpath} ({python})'
+
+    group = parser.add_argument_group('Basic options')
+    group.add_argument('-h', '--help', action='help', help='Show this help message and exit.')
+    group.add_argument('-V', '--version', action='version', help='Show version and exit.', version=message)
+
+    # ...
+    group = parser.add_argument_group('File specification',
+            description = "One of the below methods can be used to specify which files should be translated."
+            ).add_mutually_exclusive_group(required=True)
+    group.add_argument('-f', '--files', nargs='+', type=Path,
+            help="A list of files to be translated as a project.")
+    group.add_argument('-g', '--glob', type=str,
+            help=("A glob that should be used to recognise files to be translated as a project (e.g. '**/*.py'). "
+                  "Note: quote the pattern to prevent shell expansion."))
+    group.add_argument('-d', '--file-descr', type=Path,
+            help="A file where each line is the name of a file, this set of files are the files to be translated as a project.")
```

Comment:
This help message repeats the word "file" and "files", twice each. This has to be a text file, correct? And we accept ASCII or UNICODE. Both relative and absolute paths, I believe.

```suggestion
    group.add_argument('-d', '--file-descr', type=Path,
            help="A text file (ASCII or UNICODE) containing the paths to the files to be translated as a project. One path (relative or absolute) per line.")
```

Reply:
Fixed : https://github.com/pyccel/pyccel/commit/cdcb40539d8445bcad3a91f7f01a0be0610f4d94